### PR TITLE
More CODEOWNERS

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -15,14 +15,22 @@
 # Administrata and CIs
 *.toml  @larsoner
 *.yml  @larsoner
-*.rst  @larsoner
+*.rst  @larsoner @peternewman
 *.cfg  @larsoner
+codespell.1.include @peternewman
```
Comment (peternewman) [inline .github/CODEOWNERS:20]: Did you intentionally miss this one @larsoner ?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -15,14 +15,22 @@
...
 coverage*  @larsoner
-.github/*  @larsoner
-Makefile  @larsoner
+.github/*  @larsoner @peternewman
+.gitignore @peternewman
```
Comment (peternewman) [inline .github/CODEOWNERS:23]: And here?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -15,14 +15,22 @@
...
+Makefile  @larsoner @peternewman
 
 # Python code
-codespell_lib/*.py  @larsoner
+codespell_lib/*.py  @larsoner @peternewman
+bin/* @peternewman
```
Comment (peternewman) [inline .github/CODEOWNERS:28]: Or this?

File: .github/CODEOWNERS (hunk 1 above)
Comment (larsoner) [reply .github/CODEOWNERS:20]:
```suggestion
codespell.1.include @larsoner @peternewman
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (larsoner) [reply .github/CODEOWNERS:23]:
```suggestion
.gitignore @larsoner @peternewman
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (larsoner) [reply .github/CODEOWNERS:28]:
```suggestion
bin/* @larsoner @peternewman
```

**Review by larsoner — APPROVED** [review]

**Pull request merged**